src/main/kotlin/com/nikitahohulia/listeningplatform/repository/CustomUserRepository.kt

```diff
@@ -0,0 +1,19 @@
+package com.nikitahohulia.listeningplatform.repository
+
+import com.nikitahohulia.listeningplatform.entity.User
+import org.bson.types.ObjectId
+
+interface CustomUserRepository {
+
+    fun getUserById(id: ObjectId): User?
```

Comment:
```suggestion
    fun findUserById(id: ObjectId): User?
```

Reply:
@ajax-shulepa-a 
I told to our kids earlier that we use `getBy*` and `findBy*` conventions. 

I've found the description of the convention in our wiki: just search by `Core team code conventions`.
(sorry, sharing links in public repo is insecure)